src/styles/utils/_extends.scss

```
@@ -0,0 +1,10 @@
+%text-center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+%font-day-box {
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
```

Як на мене зрозуміліше коли ці властивості задані відразу в глобальних стилях body чи html. 
```suggestion
body {
  font-family: Arial, sans-serif;
  font-size: 30px;
}
```